src/gameComponents/Ship/ProtoShip.java

```diff
@@ -0,0 +1,21 @@
+package gameComponents.Ship;
+
+import BBDGameLibrary.Geometry2d.BBDPoint;
+
+/**
+ * A class to serve as a precursor to a proper ship class.  Primarily serves as a testing
+ * bed for building new objects and the like.
+ */
+public class ProtoShip {
+    private enum Size {SMALL, MEDIUM, LARGE}
```

Comment:
why private over public

Reply:
prefer private so that other classes can't change it.

We can add getters if we need to.  Mostly I want to make it very hard for an outside class to modify an object of this class's fields on accident.

public makes it way to easy

Reply:
make it final if you don't want the list to be modified, but public for access? Other things will want to be able to know what the sizes as an enum are

Reply:
Can an enum even be added to outside of its definition?

Prefer to leave it private for now.  That should resolve itself once Size is refactored to an enum.

Reply:
enums are are implicitly final